### PR TITLE
[develop] Expose and use overrideWrapDomain

### PR DESCRIPTION
**Pull request by mrmr1993**
This PR is #91 for develop. This is the counterpart to https://github.com/MinaProtocol/mina/pull/13686.

### CODE REVIEW

File: ocaml/lib/pickles_bindings.ml [hunk 1]
```diff
@@ -461,13 +463,26 @@ let pickles_compile (choices : pickles_rule_js array)
   (* translate method circuits from JS *)
   let public_input_size = signature##.publicInputSize in
   let public_output_size = signature##.publicOutputSize in
+  let override_wrap_domain =
+    match Js.Optdef.to_option signature##.overrideWrapDomain with
```
Comment (dannywillems) [inline ocaml/lib/pickles_bindings.ml:467]: Berkeley uses
```
  let override_wrap_domain =
    Js.Optdef.to_option signature##.overrideWrapDomain
    |> Option.map ~f:Pickles_base.Proofs_verified.of_int
  in
```

Comment (mitschabaude) [reply ocaml/lib/pickles_bindings.ml]: updated!

Comment (dannywillems) [reply ocaml/lib/pickles_bindings.ml]: I make a commit changing to it, feel free to remove it if you do not agree with

**Review by mitschabaude — APPROVED** [review]

File: ocaml/lib/pickles_bindings.ml (hunk 1 above)
Comment (dannywillems) [reply ocaml/lib/pickles_bindings.ml]: Too late :grimacing:. You're a more morning person than me :)

**Review by dannywillems — APPROVED** [review]

File: ocaml/lib/pickles_bindings.ml (hunk 1 above)
Comment (mitschabaude) [reply ocaml/lib/pickles_bindings.ml]: I already made that commit :D

**Pull request merged**